framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -4921,6 +4921,17 @@ VulkanReplayConsumerBase::OverrideCreateImage(PFN_vkCreateImage
         {
             image_info->queue_family_index = 0;
         }
+
```

Comment:
Can you use `GetPNextStruct` from `graphics/vulkan_util.h` to find this struct?

Reply:
Done